website/docs/components/copy/snippet/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Copy Snippet
+description: A button that copies the text content of the button itself.
```

Comment:
```suggestion
description: A button that allows users to copy the text content displayed on the button itself.
```

Reply:
What about "A button that allows users to copy a snippet of code" ?